### DISCUSSION TITLE
UI-Lovelace-Minimalist Wiki - bad links

### DISCUSSION THREAD

**Discussion by iqlogic** [UI-Lovelace-Minimalist/UI#131] [General]
Hello,

I just want to let you know that there are bad links causing 404 error. Last two links on the end of this section:
https://ui-lovelace-minimalist.netlify.app/installation#a-little-technical-insight

> Whatever way you choose, here are the instructions:
> For yaml-mode here - it should be https://ui-lovelace-minimalist.netlify.app/installation/yaml-mode
> For UI-mode here - it should be https://ui-lovelace-minimalist.netlify.app/installation/ui-mode

**Comment by iqlogic** [comment]
And also here is bad link:
https://ui-lovelace-minimalist.netlify.app/installation/yaml-mode

> Note: these are the installation instructions, if you use yaml-mode! We do not recommend UI-mode, but if you really need to go with it, than use these instructions.

Here "[these instructions](https://ui-lovelace-minimalist.netlify.app/installation/installation-ui)" should be link https://ui-lovelace-minimalist.netlify.app/installation/ui-mode

**Reply by CM000n** [reply]
Hi @iqlogic, thank you for your advice. Unfortunately, we currently have no possibility to revise anything on the wiki. :-(

See here:
https://github.com/UI-Lovelace-Minimalist/UI/discussions/100#discussioncomment-1857208

We currently have to consider next steps on how to further deal with this repo, respectively the UI organization.

**Comment by iqlogic** [comment]
@CM000n , oh I see, that's unfortunate. Well, I hope that you will gain access soon so you can continue on your amazing project 😊
